keyboards/southpole/keymaps/default/keymap.c

```diff
@@ -0,0 +1,25 @@
+#include "southpole.h"
```

Comment:
This line should be:

```suggestion
#include QMK_KEYBOARD_H
```